README.md

```diff
@@ -20,6 +22,24 @@ JS-tests are run with Shadow-cljs. Install npm, run `npm install` and execute te
 npm test
 ```
 
+## Test server
+There is a test server, that can be used as an example when developing signature solutions. 
+
+The server can be used to sign certificates together with the card reader software.
+
+### Prerequisites
+1. Install the card reader software [https://dvv.fi/en/card-reader-software]()
+1. Visit [https://localhost:53952/]() and accept non-trusted certificate.
+1. Download root certificates of citizen certificates from DVV by running:\
+`./download-dvv-trusted-certificates.sh`
+ 
+### Running the server
+1. Start the server:\
+`lein with-profile dev run -t citizen-certificate-roots.pem`
+1. Start a reverse proxy to serve the content over https:\
+`docker run -p 443:443 caddy caddy reverse-proxy --from localhost:443 --to http://host.docker.internal:3000`
```

Comment:
host.docker.internal ei mun käsittääkseni toimi kaikilla käyttöjärjestelmillä, olikohan että Linuxilla ei toimi. Jos näin on, niin olisiko tälle jokin universaali tapa? Välttämättä ei ole.